docs/command-reference/commands-details/workload_create_update_apply.md

```diff
@@ -0,0 +1,1154 @@
+# tanzu apps workload apply
+
+tanzu apps workload apply is a command used to create/update workloads that will be deployed in a cluster through a supply chain.
```

Comment:
Replace tanzu apps workload apply with `tanzu apps workload apply` . If its a command, put the command in (``)